remove-user-on-logout/src/main/java/de/intension/resources/admin/VidisAdminRealmResourceProvider.java

```diff
@@ -27,8 +27,9 @@ public class VidisAdminRealmResourceProvider
     implements AdminRealmResourceProvider
 {
 
-    private static final Logger      LOG                                       = Logger.getLogger(VidisAdminRealmResourceProvider.class);
-    private static final int         TOLERANCE_FOR_USER_IN_CREATION_IN_SECONDS = 5;
+    private static final Logger      LOG                                               = Logger.getLogger(VidisAdminRealmResourceProvider.class);
+    private static final int         DEFAULT_TOLERANCE_FOR_USER_IN_CREATION_IN_SECONDS = 30;
+    public static final String       DELETION_TOLERANCE_CONFIG                         = "deletiontolerance";
```

Comment:
shouldn't this new config param be documented? It's probably only configurable with system property / environment variable (`KC_SPI_...` right?)

Reply:
right will do.